### PR TITLE
Added historical model to course run.

### PR DESCRIPTION
**Pull request by HassanJaveed84**

### CODE REVIEW

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -731,6 +731,7 @@ class CourseRun(DraftModelMixin, PkSearchableMixin, TimeStampedModel):
 
     everything = CourseRunQuerySet.as_manager()
     objects = DraftManager.from_queryset(CourseRunQuerySet)()
+    history = HistoricalRecords(excluded_fields=['slug'])
```
Comment (pwnage101) [inline course_discovery/apps/course_metadata/models.py:734]: What's the purpose of excluding the slug field?

Comment (HassanJaveed84) [reply course_discovery/apps/course_metadata/models.py:734]: I excluded it because of https://github.com/edx/course-discovery/pull/332

**Review by pwnage101 — APPROVED** [review]
I'll base my migration on yours

**Comment by pwnage101** [comment]
@HassanJaveed84 discovery team has been notified, please merge this, then I will merge mine, and then we can double check behavior on stage.

**Pull request merged**